### PR TITLE
update broken link

### PR DESCRIPTION
**Pull request by jialaijun**
update broken link(https://github.com/jialaijun/website/blob/master/zh/docs/reference/glossary), the right link is : https://github.com/jialaijun/website/blob/master/content/zh/docs/reference/glossary

### CODE REVIEW

**Comment by tengqm** [comment]
- pls sign the CLA.
- pls propose a complete PR to the page. There are other changes to the English page which can be identified using `./scripts/lsync.sh content/zh/docs/contribute/style/hugo-shortcodes/index.md` command.

/hold

**Review by Arhell — COMMENTED** [review]
@jialaijun
Pls sign the CLA in order for the PR to be reviewed.

**Comment by tengqm** [comment]
/hold
There are other issues.

**Comment by tengqm** [comment]
The PR is not using the latest full_link, please check the latest English version.

File: content/zh/docs/reference/glossary/cri-o.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ tags:
 title: CRI-O
 id: cri-o
 date: 2019-05-14
-full_link: https://cri-o.io/docs/
+full_link: https://cri-o.io
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/cri-o.md:18]: The full_link is `https://cri-o.io/#what-is-cri-o`

Comment (tengqm) [reply content/zh/docs/reference/glossary/cri-o.md:18]: Same on line 5.

**Comment by tengqm** [comment]
Issue resolved in another PR. Thanks for the contribution anyway.
/close

**Pull request closed without merging**